mkdocs/docs/HPC/teaching_training.md

```diff
@@ -0,0 +1,139 @@
+# Teaching and training
+
+The HPC infrastructure can be used for teaching and training purposes, and
+HPC-UGent provides support for getting you organized. 
+
+As a reminder, both Bachelor and Master students are allowed to use the HPC, and it is also possible to organize workshops. But in either case we do recommend preparing a fallback plan for should the HPC infrastructure be unavailable, e.g. because of an unexpected power failure.
+
+In general, we advise the use of the HPC webportal in combination with the [interactive cluster {https://docs.hpc.ugent.be/interactive_debug/} ] for teaching and training, but deviations are possible upon request.
+
+In order to prepare things for your course, provide hpc@ugent.be with the following information (explained further below):
+
+- UGent Course Code or workshop name
+- Vsc-ids of all course group moderators
+- Start and end date (and time) for your course/workshop
+- In the case of a workshop: vsc-ids of all participants
+- Optional information
+  - special storage requirements: shared folder and/or shared subgroup folders
+  - reservation for resource requirements beyond the interactive cluster
+  - ticket number for specific software needed for your course/workshop
+  - more details for a custom Interactive Application in the webportal
+
+In addition, it could be beneficial to set up a short Teams call with HPC-UGent, especially if you are using a complex workflow for your course/workshop. 
+
+Please make these requests well in advance, several weeks before the start of your course/workshop.
+
+
+## UGent Course Code
+
+Based on the Course Code, we can create vsc accounts for all students that have officially enrolled in your UGent course (if they do not have an account already). Students will then no longer have to take steps themselves to request a vsc account. The students do need to be officially enrolled, so that they are linked to your Course Code.
```

Comment:
```suggestion
Based on the Course Code, we can create VSC accounts for all students that have officially enrolled in your UGent course (if they do not have an account already). Students will then no longer have to take steps themselves to request a VSC account. The students do need to be officially enrolled, so that they are linked to your Course Code.
```